hasura/migrations/default/1671022164553_create_table_public_locked_token_distribution_gifts/up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE "public"."locked_token_distribution_gifts" (
+  "id" bigserial NOT NULL, 
+  "locked_token_distribution_id" bigint NOT NULL, 
+  "profile_id" bigint NOT NULL, 
+  "earnings" numeric NOT NULL,
+  PRIMARY KEY ("id") , 
+  FOREIGN KEY ("locked_token_distribution_id") REFERENCES "public"."locked_token_distributions"("id") ON UPDATE cascade ON DELETE cascade, 
+  FOREIGN KEY ("profile_id") REFERENCES "public"."profiles"("id") ON UPDATE cascade ON DELETE cascade);
```

Comment:
should this be `restrict` instead of `cascade` ?

Reply:
good call